shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/model/DateComponent.kt

```diff
@@ -0,0 +1,5 @@
+package co.nimblehq.blisskmmic.domain.model
+
+data class DateComponent (
+    val timeInterval: Long
+)
```

Comment:
Just curious, is this class going to have more attributes in the future? If not, it is kinda unusual to have a class with a single attribute, and maybe we can just use **timeInterval** with `Long` type directly as where this data class is used. What do you think? 🤔

Reply:
@minhnimble Updated, with other comments.